src/background.js

```diff
@@ -41,6 +43,7 @@ if (isSecondInstance) {
     if (env.name !== 'production') {
       menus.push(devMenuTemplate);
     }
+    menus.push(prefMenu);
```

Comment:
Instead of doing this here, could you push this inside an `if (!IS_MAC)` guard inside `base_menu_template.js`. As far as I know, hiding the menu bar has no effect on macOS because the menu bar on macOS is at the top of the screen, not the application.

As an aside, I'm also wondering if any Linux users know if this has an effect on Linux.

Reply:
Also, would you mind renaming this to keep the same naming structure of the menu files (`_template.js`)?